charts/midaz/values.yaml

```
@@ -512,6 +516,200 @@ crm:
+  secrets:
+    # -- PostgreSQL password. Single-sourced from the Bitnami postgresql subchart
+    # Secret (key `password`) when internal; only set for an EXTERNAL PostgreSQL
+    # (subchart disabled / .external=true) without an existingSecret override.
+    DB_PASSWORD: ""
+    # -- API Key value (read by the app as $API_KEY when API_KEY_ENABLED=true).
+    API_KEY: ""
+    # -- Tenant Manager service API key (required when MULTI_TENANT_ENABLED=true).
+    # MULTI_TENANT_SERVICE_API_KEY: ""
+    # -- Optional Redis password for the tenant connection-pool registry.
+    # MULTI_TENANT_REDIS_PASSWORD: ""
+  # -- Existing secrets name
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)